db/migrate/20231110205748_add_more_e11y_fields_to_state_intakes.rb

```diff
@@ -0,0 +1,10 @@
+class AddMoreE11yFieldsToStateIntakes < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_ny_intakes, :eligibility_out_of_state_income, :integer, default: 0, null: false
+    add_column :state_file_ny_intakes, :eligibility_part_year_nyc_resident, :integer, default: 0, null: false
+    add_column :state_file_ny_intakes, :eligibility_withdrew_529, :integer, default: 0, null: false
```

Comment:
I added the `:eligibility_withdrew_529` column which I don't use in this PR, but will be useful for [the next eligibility story](https://www.pivotaltracker.com/story/show/186439181)

Reply:
It seems as the specs have some kind of dependency?
https://github.com/codeforamerica/vita-min/blob/ee8b6f1f5fd9ea53d06e486702dbfa9e69063205/spec/support/helpers/state_file_intake_helper.rb#L18